Apps/Common/SIMSolver.h

```diff
@@ -155,7 +155,7 @@ template<class T1> class SIMSolver : public SIMSolverStat<T1>
     this->printHeading(heading);
 
     // Solve for each time step up to final time
-    for (int iStep = 1; this->advanceStep(); iStep++)
+    for (; this->advanceStep(); )
```

Comment:
Then rather use `while (this->advanceStep())` instead

Reply:
alright. you have preferred butchered for's before but i'll gladly use while :)